tests/test_utils.py

```diff
@@ -67,6 +67,17 @@ def test_parse_real_name():
         "info": ":test",
     }
 
+    # This test is not great, but some folks have their e-mail instead of name.
+    assert parse_real_name("tester@testerson.example (:test)") == {
+        "name": "tester",
+        "info": "@testerson.example (:test)",
+    }
```

Comment:
Hmmm I would imagine the `name` should be the full email here no? I don't think the name is meaningful without the domain behind it included.

Reply:
This is a band-aid solution. Ideally folks don't put their e-mail address in the "real_name" field in BMO. Until we have some kind of validation on BMO this is pretty much only addressing a single edge case currently.